tests/python-pytest/onnx/export/onnx_backend_test.py

```diff
@@ -142,6 +146,9 @@
 
 BACKEND_TESTS.exclude('.*broadcast.*')
 BACKEND_TESTS.exclude('.*bcast.*')
+BACKEND_TESTS.exclude('.*expand.*')
```

Comment:
Can you add comment saying why we have excluded these test cases?
Do we have custom tests for these somewhere?

Reply:
Expand and LogSoftMax are not yet implemented in ONNX export. Expand is not yet implemented in ONNX import.

Tests for these operators are excluded because test_exp, test_log will include tests for these operators too, leading to failures.

Added a comment explaining this.

Reply:
It should be possible to not exclude these tests by using the right regular expressions in the include list.